### DISCUSSION TITLE
Restarting FDS Simulation

### DISCUSSION THREAD

**Discussion by SaiSantosh1824** [firemodels/fds#12319] [General]
Hello,

I have been able to successfully restart a FDS simulation by adding the &MISC RESTART=T/ line to the fds script. From the User guide, I have also added the command CLIP_RESTART_FILES=F to the &DUMP in order to prevent the data from being overwritten. 

I have (for safety) made a copy folder from my previous simulation which has completed 450s and am restarting the sim from 300s. Does this mean, the simulation data from 300-450s which has been carried over will remain or be overwritten? What will be the correct way to manage the data output from the simulation (if required) during the restart? 

I have searched everywhere but am not able to find a response to the above, please provide clarity.

Many thanks

**Comment by mcgratta** [comment]
First, the parameter `CLIP_RESTART_FILES` only applies to the comma-delimited output files like those with the suffix `_hrr.csv`, `_devc.csv`, etc. Second, by default, when you restart a simulation at 300 s, say, having previously run the case to 450 s, the previous output from 300 s to 450 s is going to be overwritten by the restarted simulation. In this way, you will have a monotonically increasing time sequence. I do not recall why we added this parameter (and I think that I did it myself). For whatever reason, this is all that it does -- if you set it to F, your restarted output files will have a non-sequential time history. I'm not sure why you would want this, especially if you have copied the original files somewhere else. I myself would want my restarted simulation to have a monotonic time sequence, which is the default behavior.

**Comment by SaiSantosh1824** [comment]
Thank you for clarifying this. I have amended my script to now not incorporate this parameter.

And just for clarification, removing this parameter now overwrites to have a monotonic time sequence from the restart of the sim, say 300s in my case. I will then have to combine the csv files to get the full data set?

**Comment by mcgratta** [comment]
No, your restarted case should have a time sequence starting at `T_BEGIN`, which is 0 by default. You  need not combine files.

**Reply by SaiSantosh1824** [reply]
Thank you for providing clarity on this topic.
